src/datumaro/plugins/transforms.py

```diff
@@ -1746,13 +1758,15 @@ def __iter__(self):
 
 class AstypeAnnotations(ItemTransform):
     """
-    Enables the conversion of annotation types for the categories and individual items within a dataset.|n
+    Converts the types of annotations within a dataset based on a specified mapping.|n
     |n
-    Based on a specified mapping, it transforms the annotation types,|n
-    changing them to 'Label' if they are categorical, and to 'Caption' if they are of type string, float, or integer.|n
+    This transform changes annotations to 'Label' if they are categorical, and to 'Caption'
+    if they are of type string, float, or integer. This is particularly useful when working
+    with tabular data that needs to be converted into a format suitable for specific machine
+    learning tasks.|n
     |n
     Examples:|n
-        - Convert type of `title` annotation|n
+        - Convert the type of a `title` annotation:|n
```

Comment:
Convert -> Converts

Reply:
Fix in [c79dae8](https://github.com/openvinotoolkit/datumaro/pull/1599/commits/c79dae8f38c555150d38887fcd08379ed5a0b5c2)